Porting.md

```diff
@@ -17,6 +15,38 @@ sudo apt-get install git gnupg flex bison gperf build-essential \
   g++-4.8-multilib
 ```
 
+#### Step 1: Installing required packages (Arch)
+
+First install this GPG key (it's needed for a PGP signature in the ncurses5-compat-libs and lib32-ncurses5-compat-libs packages)
+
+```
+gpg --recv-keys 702353E0F7E48EDB
+```
+
+Then install the lineageos-devel meta package from AUR
+
+**With yaourt**
```

Comment:
Please don't include yaourt - it has so many problems... pacaur is the most popular one right now and it works a lot better.